doc/source/rllib-algorithms.rst

```diff
@@ -422,7 +422,7 @@ Tuned examples: `CartPole-v0 <https://github.com/ray-project/ray/blob/master/rll
 
 Evolution Strategies
 --------------------
-|tensorflow|
+|pytorch| |tensorflow|
```

Comment:
Don't forget to update the sigils in rllib-toc.rst (should also add for DQN, etc. missed in the past)

Reply:
+1 Will do.